### PR TITLE
Add helper to run a query and select the response

### PR DESCRIPTION
**Pull request by jaresty**

### CODE REVIEW

File: GPT/gpt.py [hunk 1]
```diff
@@ -110,6 +110,14 @@ def paste_model_confirmation_gui():
         GuiState.text_to_confirm = ""
         confirmation_gui.hide()
 
+    def paste_selected(result: str):
```
Comment (C-Loftus) [inline GPT/gpt.py:113]: Isn't `paste_and_select` perhaps more clear?

**Comment by C-Loftus** [comment]
This looks like a good idea. I like how it mimics statefulness without needing to explicitly add state internally. I made a few comments then I think we can merge?

File: GPT/gpt.py (hunk 1 above)
Comment (jaresty) [reply GPT/gpt.py]: Works for me 👍

File: GPT/gpt.talon [hunk 2]
```diff
@@ -15,6 +15,12 @@ model clip <user.modelPrompt> [this]$:
     result = user.gpt_apply_prompt(modelPrompt, text)
     clip.set_text(result)
 
+# Runs a model prompt on the selected text and sets the selection to the pasted response
+model selected <user.modelPrompt> [this]$:
```
[the remark this answers is not in the document]
Comment (jaresty) [reply GPT/gpt.talon]: Sounds good. Changed ✅

**Comment by C-Loftus** [comment]
Thanks for making edits. I will test this out locally later/tomorrow, perhaps add some docs, change the old clip command, then merge

**Comment by C-Loftus** [comment]
Tested. Merging this! Thanks for this idea, I think it will be useful! I am curious to continue thinking about other results / pipelines for model outputs (i.e. we have select, clip, and I wonder if there are other ways to process that would be useful. Will continue to brainstorm for future PRs)

**Pull request merged**